resources/definitions/fdmprinter.def.json

```diff
@@ -3381,27 +3381,45 @@
                 },
                 "support_skip_some_zags":
                 {
-                    "label": "Skip Some ZigZags Connections",
-                    "description": "Skip some ZigZags connections to make the support structure easier to break.",
+                    "label": "Break Up Support In Chunks",
+                    "description": "Skip some support line connections to make the support structure easier to break away. This setting is applicable to the Zig Zag support infill pattern.",
                     "type": "bool",
                     "default_value": false,
                     "enabled": "support_enable and (support_pattern == 'zigzag')",
                     "limit_to_extruder": "support_infill_extruder_nr",
                     "settable_per_mesh": false,
                     "settable_per_extruder": true
                 },
-                "support_zag_skip_count":
+                "support_skip_zag_per_mm":
                 {
-                    "label": "ZigZag Connection Skip Count",
-                    "description": "Skip one in every N connection lines to make the support structure easier to break.",
-                    "type": "int",
-                    "default_value": 6,
-                    "minimum_value": "1",
-                    "minimum_value_warning": "3",
+                    "label": "Support Chunk Size",
+                    "description": "To skip one connection line in every N mm to make the support structure easier to break away.",
+                    "type": "float",
+                    "unit": "mm",
+                    "value": "support_line_distance * 5",
```

Comment:
I think a static value makes more sense here.
Maybe a default value of 10 mm would be better.